.travis.yml

```
@@ -44,10 +52,13 @@ jobs:
       script:
         - make test-coverage
         - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage-all.out -repotoken=$COVERALLS_TOKEN
-
+      after_success:
+        - echo "Coverage succeeded, see in goveralls if the % of code covered with tests decrease."
```

```suggestion
      after_success: echo "Coverage succeeded, see in goveralls if the % of code covered with tests decreased."
```

One line and `s/decrease/decreased`